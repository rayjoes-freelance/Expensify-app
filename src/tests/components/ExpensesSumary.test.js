import ExpenseSummary from '../../components/ExpensesSummary';
import React from 'react';
import {shallow} from 'enzyme';
import expenses from '../fixtures/expensesArray';


test("Should count the good number of expenses", () => {
    const wrapper = shallow(<ExpenseSummary expenses={expenses}/>);
    expect(wrapper).toMatchSnapshot();
});

