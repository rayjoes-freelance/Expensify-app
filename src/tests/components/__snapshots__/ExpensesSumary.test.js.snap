// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should count the good number of expenses 1`] = `
<div>
  <h4>
    Viewing 3 expenses totalling of $1,141.96
  </h4>
</div>
`;
